modules/core/src/model/PurposeRestrictionVector.ts

```diff
@@ -148,11 +148,13 @@ export class PurposeRestrictionVector extends Cloneable<PurposeRestrictionVector
 
     })();
 
+    const values = [...Array(lastEntry).keys()].map( (i) => i + 1);
```

Comment:
There should be a comment here as to why `1` is added to each value.

Reply:
comment added, thanks